app/models/meetup.parsing.ts

```diff
@@ -0,0 +1,57 @@
+import type { InferType } from "yup";
+import { object, string, number, array, date } from "yup";
+
+const MEETUP_EVENT_SCHEMA = object({
+  title: string().required(),
+  shortUrl: string().required().url(),
+  venue: object({
+    name: string().required(),
+    address: string().defined().strict(true),
+    city: string().defined().strict(true),
+    state: string().defined().strict(true),
+  }).required(),
+  dateTime: date().required(),
+  going: number().required(),
+}).required();
```

Comment:
Thanks so much for adding parsing/a schema! I'm a little partial to `zod`, but since you're the one doing the work I'm happy with this 😄 

It's now on me to give a good reason to use `zod` instead in the future, so I'll definitely tag you if I get around to that